app/page.tsx

```diff
@@ -148,6 +149,16 @@ export default async function Home() {
               )}
             </div>
           </div>
+          <div className="flex flex-col w-1/2 rounded-md p-4 mb-4 min-h-1/2 h-fit text-background bg-white/90">
```

Comment:
`w-1/2` bad for responsive design, I'll make this proper if we decide it's the direction we want to go in.